hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/SparkParquetReaderBase.scala

```diff
@@ -72,10 +72,6 @@ abstract class SparkParquetReaderBase(enableVectorizedReader: Boolean,
     conf.setBoolean(SQLConf.PARQUET_INT96_AS_TIMESTAMP.key, true)
     // Using string value of this conf to preserve compatibility across spark versions.
     conf.setBoolean(SQLConf.LEGACY_PARQUET_NANOS_AS_LONG.key, false)
-    if (HoodieSparkUtils.gteqSpark3_4) {
-      // PARQUET_INFER_TIMESTAMP_NTZ_ENABLED is required from Spark 3.4.0 or above
-      conf.setBoolean("spark.sql.parquet.inferTimestampNTZ.enabled", false)
-    }
```

Comment:
To discuss with @yihua

Reply:
create another oss pr and remove just this and see what tests fail

Reply:
Is this still needed?

Reply:
instead of deleting it I just set it to true. Maybe instead I should just set it to true if the config is unset?

Reply:
Yes, that works.